mmv1/third_party/terraform/services/netapp/resource_netapp_volume_test.go

```diff
@@ -318,8 +318,21 @@ resource "google_netapp_volume" "test_volume" {
 			nfsv4                 = false
 		}
 	}
+	# Delete protection only gets active after an NFS client mounts.
+	# Setting it here is save, volume can still be deleted.
 	restricted_actions = ["DELETE"]
-	}
+	deletion_policy = "FORCE"
+}
+
+# Add the following snapshot block to the test as soon as snapshot resoruce
+# is added to the provider. It will make the test cleanup require
+# deletion_policy = "FORCE" on the volume for successful delete.
+# resource "google_netapp_volumesnapshot" "test-snapshot" {
```

Comment:
Honestly I'm not 100% sure if this will test the force deletion with the snapshot created within the same config, since Terraform will delete the resources by orders, snapshot first and then volume. However, that's more like a testing on the API functionality and it should be sufficient to just check if `deletion_policy` can be set correctly and the delete url is updated accordingly for this feature in Terraform. That said - we can remove this google_netapp_volumesnapshot config block IMO.

Reply:
You are correct. Looks like I was already tired when I wrote this :)